docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -296,7 +296,9 @@ The master supports the following configuration settings:
 
       -  ``user_name``: The username that the Launcher will run as. It is recommended to set this to
          something other than ``root``. The user must have a home directory with read permissions
-         for all users to enable access to generated ``sbatch`` scripts and job log files.
+         for all users to enable access to generated ``sbatch`` scripts and job log files. It must
+         have access to the Slurm/PBS queue and node status commands (``squeue``, ``sinfo``,
+         ``pbsnodes``, ``qstat`` ) to discover partitions and to display cluster usage.
```

Comment:
sbatch & qsub too.

Reply:
Those are done via setuid as the launching user not as this user_name which is what we use for for monitoring so I explicitly left them out.    They are covered in the HPC Launching Architecture page though.

Vishnu configure the launcher as user nobody which didn't work too well :-), so was attempting to clarify that.